magenta/music/musicxml_parser.py

```diff
@@ -410,7 +410,8 @@ def _parse_attributes(self, xml_attributes):
       elif child.tag == 'transpose':
         transpose = int(child.find('chromatic').text)
         self.state.transpose = transpose
-        self.key_signature.key += transpose
+        if self.key_signature != None:
```

Comment:
We prefer `if self.key_signature is not None`

https://google.github.io/styleguide/pyguide.html?showone=True/False_evaluations#True/False_evaluations